### PR TITLE
fix(quicksearch): 2513 unhighlight, open on enter

### PR DESCRIPTION
**Pull request by cy**
- remove ctrl+space listener (didn't work)
Closes #2513

### CODE REVIEW

**Comment by alex-wilmer** [comment]
👍

File: app/scripts/components/quick-search/quick-search.directives.ts [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-    Down = 40,
-    Tab = 9
-  }
 
   /* @ngInject */
-  function QuickSearch($uibModal: any, $window: ng.IWindowService, $uibModalStack): ng.IDirective {
+  function QuickSearch($uibModal: any,
+                       $window: ng.IWindowService,
+                       $uibModalStack,
+                       KEY_CODE: Object
```
Comment (shanewilson) [inline app/scripts/components/quick-search/quick-search.directives.ts]: doesn't this lose autocomplete?

Comment (cy) [inline app/scripts/components/quick-search/quick-search.directives.ts]: yeah, but the tests pass and KeyCodes are only declared once. Tradeoff?

Comment (shanewilson) [inline app/scripts/components/quick-search/quick-search.directives.ts]: I think I would rather have duplicate code with typesafety/autocompletion

**Comment by cy** [comment]
So all I did was reset to the commit where importing enums was used, and now Phantom's ok with it and the tests pass. Hooray, no duplicate code and autocompletion.

**Comment by shanewilson** [comment]
👍

**Pull request merged**